### ISSUE TITLE
Can't install on OS X Sierra

### ISSUE THREAD

**Issue by xird** [IJHack/QtPass#337]
I'm trying to install QtPass on OS X 10.12.6 according to the instructions on the main site. I've installed the GPGSuite from gpgtools.org. I've also installed pass successfully, and it works fine from the command line.

I've also installed qtpass itself just fine with:

$ brew cask install qtpass

..but I'm unable to install the pinentry program. I'm trying to follow the steps under "Dependencies" on the main site front page. The first commands work fine:

$ brew install qt5
$ brew link --overwrite --force qt5

..but the next one fails:
$ xcode-select --install
xcode-select: error: command line tools are already installed, use "Software Update" to install updates

Assuming that that means I've already done what needed to be done in that stage, I tried the next command with no more success:

$  qmake && make && macdeployqt QtPass.app
Usage: qmake [mode] [options] [files]
⋮
lots of instructions
⋮

I can launch QtPass from Applications, and it can see the files in my .password-store, but clicking on Users gives "Unable to get list of available gpg keys". In Config->Programs I've selected "Use pass".

Any ideas?

**Comment by annejan** [comment]
The pinentry currently only works on macos when using the https://gpgtools.org/ suite.

**Comment by xird** [comment]
Ok, I think I mixed up things after repeatedly installing and uninstalling things. So the "dependencies" part of the instructions refers to qt, which obviously works already, since I can launch qtpass. So the CLI error messages are not relevant?

The pinentry program should also be installed, since I have installed GPGSuite, like I mentioned.

I'm actually guessing that I never get to the pinentry stage, because of the error message: "Unable to get list of available gpg keys". I should be able to get the _list_ of keys without decrypting anything, right? After all, I can do "gpg --list-keys" on the CLI just fine and without entering a passphrase.
